### DISCUSSION TITLE
Migaku Japanese Dictionary

### DISCUSSION THREAD

**Discussion by GriefBacon** [MikeMoolenaar/Migaku-Japanese-Addon-Updated#1] [Ideas]
I know I'm not the only Japanese learner who would love to see the same treatment for the Migaku dictionary. If you have time, please consider updating it - it would be highly appreciated.

Otherwise thanks for the Japanese Addon Update, this is amazing :D

**Comment by MikeMoolenaar** [comment]
Hey, I'm currently not considering to pickup the Migaku dictonary addon. I only used the Forvo and Google Images features and have found replacements for them that work with Anki 2.1.50+ (like, [anki-forvo-dl](https://ankiweb.net/shared/info/858591644)). I couldn't find a good replacements for the Japanese addon, so I decided to fork and update it.   
I might reconsider picking up the dictonary addon if enough people show interest (by upvoting this idea, for example). I do hope you have found some replacement, if you do please let me know here.  

As for your word of thanks, you are very welcome! I'm happy at least someone found it useful.
